dashboards-observability/public/components/app.tsx

```diff
@@ -4,20 +4,21 @@
  */
 
 import { I18nProvider } from '@osd/i18n/react';
+import { QueryManager } from 'common/query_manager';
```

Comment:
Was 'common/query_manager' instead of './common/query_manager' working for you? I've seen build errors not sure if it's only on my side or not.

Reply:
this is from tsserver's `_typescript.organizeImports` command reordering lines, it shouldn't make any difference functionally